<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stereogramm 2.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: #333</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Nah-Clipping</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="nearSlider">
          <property name="maximum">
           <number>254</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Fern-Clipping</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="farSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="value">
           <number>160</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Bildüberlagerung (Tiefenbild) [%]</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="overlayDepthSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fitFrameCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Tiefenbild in Kamerabild einpassen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Augenabstand [mm]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="eyeDistanceSpinBox">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="minimum">
           <number>25</number>
          </property>
          <property name="value">
           <number>67</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>µ [0..1]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="muSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>333</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Auflösung [DPI]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="resolutionSpinBox">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="value">
           <number>94</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Kinectneigung</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tiltSpinBox">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="minimum">
           <number>-27</number>
          </property>
          <property name="maximum">
           <number>27</number>
          </property>
          <property name="singleStep">
           <number>3</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Muster</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeComboBox">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>Auflösung</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stereogramSizeComboBox">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="freezePushButton">
            <property name="styleSheet">
             <string notr="true">color: black; background-color: white</string>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder></zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOpenTexture"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDepthImageAs"/>
    <addaction name="actionSaveStereogramAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Fenster</string>
    </property>
    <addaction name="actionDetachStereogram"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionPrintAutostereogram"/>
    <addaction name="actionResetFileSequenceCounter"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuWindow"/>
   <addaction name="menuExtras"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenTexture">
   <property name="text">
    <string>Textur öffnen ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionFreeze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Freeze (Ctrl+F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSaveDepthImageAs">
   <property name="text">
    <string>Tiefenbild speichern als ... (Strg+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveStereogramAs">
   <property name="text">
    <string>Stereogramm speichern als ... (Umsch+Strg+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionDetachStereogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stereogramm &amp;aushängen (Strg+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionPrintAutostereogram">
   <property name="text">
    <string>Autostereogram drucken (Strg+P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionResetFileSequenceCounter">
   <property name="text">
    <string>Zähler für automatischen Speichern zurücksetzen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
